content/knowledge-base/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Knowledge Base
+weight: -1
+cascade:
+    version: "0"
+    docs: false
+    product: "Knowledge Base"
+---
+
+{{< img src="media/banner.png" alt="Crossplane Popsicle Truck" eager=true >}}
+
+The Crossplane Knowledge Base is a collection of documents covering recommended
+practices, third-party integrations and Crossplane-related how-tos. 
```

Comment:
![CleanShot 2023-01-24 at 15 14 32@2x](https://user-images.githubusercontent.com/492942/214304696-5559d193-efa6-4233-8793-f3a2a7349fd2.png)

Needs some more spacing.

Reply:
Fixed with [8b0347c](https://github.com/crossplane/docs/pull/271/commits/8b0347c35ef9c6a1ff3b6daa0469f994e3ee1133). I'm going to leave the spacing up to the author and not automatically put padding under the image. 

![Screen Shot 2023-01-24 at 9 35 21 AM](https://user-images.githubusercontent.com/10537576/214323199-0e773d20-7dfa-4bff-8827-3105abe723f9.png)